### PR TITLE
Helper class for page breaks inside elements

### PR DESCRIPTION
**Pull request by simshaun**
See #9543

### CODE REVIEW

File: scss/typography/_print.scss [hunk 1]
```diff
@@ -77,5 +77,10 @@ $print-hrefs: true !default;
     // Avoid page breaks after a heading
     h2,
     h3 { page-break-after: avoid; }
+
+    // Helper to re-allow page breaks in the middle of certain elements (e.g. pre, blockquote, tr)
+    .print-break-inside {
```
Comment (ncoden) [inline scss/typography/_print.scss]: `.print-break-inside` vs `.print-page-break` (I prefer this) vs `.print-page-break-inside` ?

Comment (rafibomb) [inline scss/typography/_print.scss]: Yeah - i vote `.print-page-break`

Comment (simshaun) [inline scss/typography/_print.scss]: `.print-page-break` makes me think I can use that on an element anywhere I want a page break in print (e.g. in-between two <p>s), but that's not what this feature does.

`.print-break-inside` on a containing element makes more sense to me, semantically.

**Review by ncoden — APPROVED** [review]

**Comment by ncoden** [comment]
I understand why you opened this PR for `6.4` (this is a new feature), but be advised it probably won't be released before May.

**Comment by simshaun** [comment]
That's fine. I'll add it to our styles & make a note to remove when 6.4 is released.  Thank you.

**Pull request merged**